infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SenderBufferManager.java

```diff
@@ -260,6 +302,26 @@ public LogReplicationEntryMsg resend() {
         return ack;
     }
 
+    private void activateBackpressure() {
```

Comment:
these 2 methods are for metrics only.  Can you rename them to reflect the same?  Something like logBackPressureActivationMetrics() or something similar.

Reply:
In addition to the metrics they also flip that boolean field to indicate when the timeouts should continue to be extended